test/parallel/test-tls-getprotocol.js

```diff
@@ -34,7 +34,7 @@ const server = tls.createServer(serverConfig, common.mustCall(function() {
       secureProtocol: v.secureProtocol
     }, common.mustCall(function() {
       assert.strictEqual(this.getProtocol(), v.version);
-      this.on('end', common.mustCall(function() {
```

Comment:
is `end` not being emitted anymore?

Reply:
It is, howecer the teardown (which the test inside depends on) now happens in a different tick (https://github.com/nodejs/node/pull/31806/files/6e8b7ca62a1d5dcd84918a870f57dd35a2869a70#r380861047). I think it is more correct to do the assertion in 'close'.

Reply:
Changed this the other way around. Will see if that's better or not. Will make a note once this PR passes CI.

Reply:
can you add some assertion on ‘end’ as well?

Reply:
What kind of assertion did you have in mind? Just `mustCall`?

Reply:
yes, exactly.

Reply:
yes, exactly.

Reply:
Added here and in the other location as well.